daemon/stop.go

```diff
@@ -38,9 +38,10 @@ func (daemon *Daemon) containerStop(container *container.Container, seconds int)
 		return nil
 	}
 
+	stopSignal := container.StopSignal()
 	// 1. Send a SIGTERM
-	if err := daemon.killPossiblyDeadProcess(container, container.StopSignal()); err != nil {
-		logrus.Infof("Failed to send SIGTERM to the process, force killing")
+	if err := daemon.killPossiblyDeadProcess(container, stopSignal); err != nil {
+		logrus.Infof("Failed to send signal %d to the process, force killing", stopSignal)
```

Comment:
I wonder; should we show the number here, or try to translate it to a "named" signal using https://github.com/docker/docker/blob/729c9a97822ebee2c978a322d37060454af6bc66/pkg/signal/signal_linux.go#L8 (and if the signal is not in there, show the number)